introduction.md

```diff
@@ -0,0 +1,3 @@
+Hello, it is for Mastering Github course in codeschool ;) 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.